torch/csrc/jit/codegen/cuda/scheduler/reduction.cpp

```diff
@@ -387,37 +387,38 @@ ReductionParams innerReductionHeuristic(
 
   if (rparams.cross_grid_inner_reduce) {
     rparams.grid_dim_inner_reduction = ParallelType::BIDx;
-    gdimx = gridim;
```

Comment:
Actively working on these heuristics for cross-block persistent kernels. This file can mostly be ignored.